30413/Stefan Alexandru Cimpan/Lab03/README.md

```diff
@@ -1,5 +1,11 @@
-# Lab X
-
-> Edit this file and describe in a few words what you've done in this lab.
+# Lab 3
 
+> sooo, till now I:
+learned:
+-how to make a singly linked list (def, etc)
+-how to make a doubly linked list (def, parcurgerea, finding a node, inserting a node etc)
+-how to insert in front of a list
+-purging a list
 
+-I tried my best to do the mandatory problem, I am getting closer and closer to getting those problems done correctly
+also destroyed my mental health trying to debbug my code, ty for reading and have a good day :)
```

Comment:
Trying again and again is the path to success. Never give up on it. This is how you learn to debug and write code.